packages/vim-full/build.sh

```diff
@@ -1,6 +1,7 @@
 TERMUX_PKG_HOMEPAGE=http://www.vim.org/
 TERMUX_PKG_DESCRIPTION="Vi IMproved - enhanced vi editor"
-TERMUX_PKG_DEPENDS="ncurses, vim-runtime, python, perl, ruby, lua, liblua, tcl"
+TERMUX_PKG_DEPENDS="ncurses, vim-runtime"
+TERMUX_PKG_SUGGESTS="python, perl, ruby, lua, liblua, tcl"
 TERMUX_PKG_BUILD_DEPENDS="ruby-dev, liblua-dev, tcl-dev"
```

Comment:
`TERMUX_PKG_BUILD_DEPENDS` should probably be the same list as `TERMUX_PKG_SUGGESTS`? (At least when you are trying to build with the Termux builds of them.)

No idea if `lua` or `liblua` should be in Suggests but you don't need to list any subpackage (i.e. lua and the *-dev packages) in TERMUX_PKG_BUILD_DEPENDS.

Reply:
dev packages actually should go into BUILD_DEPENDS, it's meant to make it possible to speed up builds by downloading dependencies instead of building them, see https://github.com/termux/termux-packages/pull/1169.

Reply:
I don't think we need to consider the not-yet-merged PR for now, because we would want to make any required changes in a batch (with a script?) anyway. Also we don't know for sure if listing -dev is going to be the ultimate solution? (Maybe we would hack build-order.py or so?) For one -dev packages doesn't inheritly pull all other subpackages either.

For now consistency prevails, as it makes a batch job easier.